src/components/OcSearchBar.vue

```diff
@@ -149,6 +143,13 @@ export default {
       required: false,
       default: false,
     },
+    /**
+     * The aria label for the loading spinner
+     */
+    loadingAccessibleLabel: {
+      type: String,
+      required: true,
```

Comment:
Thinking about it, this is a breaking change. We have `$gettext` available. Could you check if you can set a sensible default in a callback?

Reply:
Please check again. `$gettext` cannot be used as default value (at least I didn't find a way to do so), thus I had to implement a computed property in addition.

Reply:
Ok, that's fine. But please use `loadingAccessibleLabel` for the prop and define another name for the computed property. The props are the public API of the component and should have good names. The names for the computed properties are not that important (of course still for us as developers of the design system ;-) ).